data.json

```diff
@@ -1,11 +1,11 @@
 {
   "data": [
     {
-      "github": "srashtisj",
-      "firstname": "Srashti",
-      "lastname": "Jain",
-      "about": "Founder TechVraksh | Google Developer Expert (Angular)",
-      "image": "https://github.com/srashtisj.png"
+      "github": "darshanip",
+      "firstname": "Darshan",
+      "lastname": "Patidar",
+      "about": "System Engineer at TCS | Cloud | CyberSec",
+      "image": "https://github.com/darshanip.png"
     }
```

Comment:
Hi @darshanip, seems you replaced an existing entry. Could you please add a new entry like this

```
{
  "data": [
     {
          "github": "srashtisj",
           "firstname": "Srashti",
           "lastname": "Jain",
           "about": "Founder TechVraksh | Google Developer Expert (Angular)",
           "image": "https://github.com/srashtisj.png"
     },
     {
           "github": "darshanip",
           "firstname": "Darshan",
           "lastname": "Patidar",
           "about": "System Engineer at TCS | Cloud | CyberSec",
           "image": "https://github.com/darshanip.png"
      }
  ]
}
```

Reply:
Second commit does have existing entry.

Reply:
Nice, thanks!